lib/sles4sap.pm

```diff
@@ -297,13 +311,14 @@ Croaks on failure.
 
 sub prepare_profile {
     my ($self, $profile) = @_;
-    return unless ($profile eq 'HANA' or $profile eq 'NETWEAVER');
+    my @valid_profiles = qw(HANA NETWEAVER SAP-ASE);
+    return unless (grep /^$profile$/, @valid_profiles);
 
     # Will prepare system with saptune only if it's available.
     my $has_saptune = $self->is_saptune_installed();
 
     if ($has_saptune) {
-        assert_script_run "saptune daemon start";
+        assert_script_run 'saptune service takeover';
```

Comment:
You should run saptune VRs too, if you are changing saptune code.

Reply:
This part of the code is less about testing `saptune`, and more about tuning the SUT with `saptune` to the proposed solution. I added VRs for ASE, HANA and NetWeaver which are at the moment the valid options. Will add a `saptune` test however to be on the safe side.